### ISSUE TITLE
Composite primary key breaks if null

### ISSUE THREAD

**Issue by aldendaniels** [ForestAdmin/forest-express-sequelize#124]
A `null` value in a composite primary key breaks detail view in forest. Forest correctly creates the url:

```
record/451836/fielda-fieldb-null/details
```

But *forest-express-sequelize* gives the following error:
```
Unexpected error: invalid input syntax for integer: "null"
```

**Comment by arnaudbesnier** [comment]
Hi @aldendaniels,

I guess you would expect that:
```
record/451836/fielda-fieldb-null/details
```
to be
```
record/451836/fielda-fieldb-/details
```

right?

**Comment by aldendaniels** [comment]
@arnaudbesnier I mainly expect to be able to click on the record in forest and see the detail view 🙃 

The issue is just the incompatibility between `forest` and `forest-express-sequelize` in this scenario. Putting `null` in the URL seems reasonable AFAICT.

**Comment by arnaudbesnier** [comment]
We'll add an issue to fix in our next sprints.
I'll let you know once fixed.

Thank you for the feedback!

**Comment by arnaudbesnier** [comment]
@aldendaniels we've investigated the issue you have. One thing that we'd like to know is your database type (Postgres, MySQL, MSSQL, ...).
Actually "real" composite primary keys cannot contain null values (at least on Postgres and MySQL).

Maybe you've defined some primary keys in you Sequelize model that are a primary key column in the database. Am I wrong?

**Comment by aldendaniels** [comment]
@arnaudbesnier I'm using Postgres. You appear to be correct! The column was marked as a primary key in the ORM, but it's not actually a primary key. Thanks for the help!

I created the model using *lumber*, which seems to have incorrectly identified the primary keys, but that would be a separate issue

**Comment by arnaudbesnier** [comment]
Yes it is a know issue about Lumber: https://github.com/ForestAdmin/lumber/issues/85.

**Comment by arnaudbesnier** [comment]
We have a added some code for protection in case of database/ORM different PK configuration.

**Comment by lucasscariot** [comment]
Fixed on the version `1.3.5`
